tests/__init__.py

```diff
@@ -8,7 +8,7 @@
 def test_version_matches():
     dist = pkg_resources.get_distribution('ics')
     print(repr(dist), dist.__dict__, sys.path, ics.__path__)
-    assert len(ics.__path__) == 1
+    assert len(ics.__path__) is 1
```

Comment:
Comparing integers with the identity is a no-go. It might work on the current implementation of  cPython with small numbers might work for now but does not work for bigger integer and is not guaranteed to work over time or with other implementation of Python (think pypy)

https://stackoverflow.com/a/306353

Reply:
Oh! I agree with this!
I am reverting this change.